promgen/static/js/promgen.vue.js

```diff
@@ -27,8 +27,24 @@ const silenceStore = Vue.reactive({
     },
     setLabels(labels) {
         this.state.labels = { ...labels };
+        for (const [key, value] of Object.entries(this.state.labels)) {
+            if (!Array.isArray(value)) {
+                if (value.includes("*")) {
+                    this.state.labels[key] = [value, '=~'];
```

Comment:
That `'=~'` can be wrapped in double quotes. There are other places with this kind of thing.

Could you try passing the new JavaScript code you are adding through Prettier? It will fix all those single quotes, missing semi-colons, indentations and other stuff for you.